.github/release_template.md

```diff
@@ -72,6 +72,7 @@ __TODO__:
 ### Final Steps
 
 * [ ] QA: [Smoke test](https://confluence.owncloud.com/display/OG/Manual+Tests#ManualTests-SmokeTest)
+* [ ] QA: Check the linux download pages whether the URLs are correct (e.g. contain stable, not testing)
```

Comment:
We need to do this after "publish the release on GitHub", which includes "moving" the release from testing/ to stable/. How about moving this to "Infrastructure check"?